docs/worker-node/install-singularity.md

```diff
@@ -193,7 +202,7 @@ To install singularity as `setuid`, make sure that your host is up to date befor
 !!! tip
```

Comment:
This tip is just repeating what you put into lines 190 to 192 ("If you are installing singularity on a worker node...")

Reply:
Yes, but it explains why.  I'm pretty sure this was Brian Bockelman's suggestion.  Anyway with singularity-3 this will go away, there's only one rpm.